migration_original/ODS1Stage/tables/Show/SOLRProvider/spu_translated_SOLRProvider.sql

```diff
@@ -175,20 +175,12 @@ declare
     update_statement_20 string;
     temp_table_statement_1 string;
     update_statement_temp_1 string;
+    update_statement_temp_2 string;
```

Comment:
remove this from this pr this is solrprovider

Reply:
Ya, no se como se colo eso ahi